src/agent/v8/legacy-debugapi.ts

```diff
@@ -194,26 +194,22 @@ export class V8DebugApi implements debugapi.DebugApi {
     let timesliceEnd = Date.now() + 1000;
     // TODO: Determine why the Error argument is not used.
     const listener =
-        this.onBreakpointHit.bind(this, breakpoint, function(err: Error) {
+        this.onBreakpointHit.bind(this, breakpoint, (err: Error) => {
           const currTime = Date.now();
           if (currTime > timesliceEnd) {
             logsThisSecond = 0;
             timesliceEnd = currTime + 1000;
           }
           print(
-              // TODO: Address the case where `breakpoint.logMessageFormat` is
```

Comment:
Is this TODO no longer relevant? It seems we're still ignoring that case.

Reply:
Good catch.  I've added the comment back.